Common/Properties/SharedAssemblyInfo.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using System.Reflection;
+using System.Runtime.CompilerServices;
 
 // common assembly attributes
 [assembly: AssemblyDescription("Lean Algorithmic Trading Engine - QuantConnect.com")]
 [assembly: AssemblyCopyright("Copyright ©  2015")]
 [assembly: AssemblyCompany("QuantConnect")]
 [assembly: AssemblyTrademark("QuantConnect")]
 [assembly: AssemblyVersion("2.2.0.2")]
+
+[assembly: InternalsVisibleTo("QuantConnect.Tests")]
```

Comment:
If we need access to internal members in order to perform testing, I think that means we need to rethink the design and break things into smaller, testable pieces as needed.